### DISCUSSION TITLE
Git hub activity overview issue

### DISCUSSION THREAD

**Discussion by Sagar9980** [community/community#43803] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi, 
I am having an issue with my profile's activity overview. I noticed that I am not able to see any commits that have been updated since December 25th. I have checked my repository and can confirm that I have made several commits since that date, but they are not appearing in my activity overview though they present in repository. I have checked email in  git installed on my local machine. There is no mismatch of email though.
 
I have tried refreshing the page and logging out and logging back in, but the issue persists. Is there anything that I can do to fix this issue or any further troubleshooting steps that you can suggest?

Thank you.

**Comment by radar-blip** [comment]
If your commits were made in a private repository, there are settings for that.

**Here**:
![contributes](https://user-images.githubusercontent.com/39196767/211464124-46e81000-9465-4c0d-80fa-7527781dbf7e.png)

**and here**:
![contributes1](https://user-images.githubusercontent.com/39196767/211464206-ac3d75ea-dd7c-4e46-8b80-23a0ad417230.png)

**Comment by Sagar9980** [comment]
@linuxlawson Thanks for your reply. And I have noticed now the issue is with branches. Actually overview and commit map  is only recording commits for the default branch not for new branch I have created. I merged my branch with default branch. It worked as i wanted. But I there any way to make this possible to update overview and commit map with commits of other branches?

**Reply by radar-blip** [reply]
According to [this article](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#contributions-that-are-counted) commits _must_ be made in the default branch.

**Reply by Sagar9980** [reply]
@linuxlawson  Thanks I got it.
